src/app.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Form from './form'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      question: '',
+      answer: ''
+    }
+    this.handleChange = this.handleChange.bind(this)
```

Comment:
This `handleChange` method should be inside your form component instead of the `App`.